client/me/security-2fa-backup-codes-list/index.jsx

```diff
@@ -213,19 +289,63 @@ module.exports = React.createClass( {
 						}.bind( this ) }
 						disabled={ this.getSubmitDisabled() }
 					>
-						{ this.translate( 'All Finished!', { context: 'The user presses the All Finished button at the end of Two-Step setup.' } ) }
-					</FormButton>
-
-					<FormButton
-						className="security-2fa-backup-codes-list__print"
-						isPrimary={ false }
-						onClick={ function( event ) {
-							analytics.ga.recordEvent( 'Me', 'Clicked On 2fa Print Backup Codes Button' );
-							this.onPrint( event );
-						}.bind( this ) }
-					>
-						{ this.translate( 'Print', { context: 'The user presses the print button during Two-Step setup to print their backup codes.' } ) }
+						{
+							this.translate( 'All Finished!',
+							{ context: 'The user presses the All Finished button at the end of Two-Step setup.' } )
+						}
 					</FormButton>
+					<ButtonGroup className="security-2fa-backup-codes-list__btn-group">
+						<Button
+							className="security-2fa-backup-codes-list__copy"
+							disabled={ ! this.props.backupCodes.length }
+							onMouseEnter={ this.enableCopyCodesTooltip }
+							onMouseLeave={ this.disableCopyCodesTooltip }
+							ref="copyCodesBtn"
+						>
+							<Gridicon icon="clipboard" />
+							<Tooltip
+								context={ this.refs && this.refs.copyCodesBtn }
+								isVisible={ this.state.copyCodesTooltip }
+								position="top"
+							>
+								{ this.translate( 'Copy Codes' ) }
+							</Tooltip>
+						</Button>
+
+						<Button className="security-2fa-backup-codes-list__print"
+							disabled={ ! this.props.backupCodes.length }
+							onClick={ this.onPrint }
+							onMouseEnter={ this.enablePrintCodesTooltip }
+							onMouseLeave={ this.disablePrintCodesTooltip }
+							ref="printCodesBtn"
+						>
+							<Gridicon icon="print" />
+							<Tooltip
+								context={ this.refs && this.refs.printCodesBtn }
+								isVisible={ this.state.printCodesTooltip }
+								position="top"
+							>
+								{ this.translate( 'Print Codes' ) }
+							</Tooltip>
+						</Button>
+
+						<Button className="security-2fa-backup-codes-list__download"
+							disabled={ ! this.props.backupCodes.length }
+							onClick={ this.saveCodesToFile }
+							onMouseEnter={ this.enableDownloadCodesTooltip }
+							onMouseLeave={ this.disableDownloadCodesTooltip }
+							ref="downloadCodesBtn"
+						>
+							<Gridicon icon="cloud-download" />
+							<Tooltip
+								context={ this.refs && this.refs.downloadCodesBtn }
+								isVisible={ this.state.downloadCodesTooltip }
+								position="top"
+							>
+								{ this.translate( 'Download Codes' ) }
```

Comment:
Alternate string suggestion: 
* `translate( 'Share embed codes and download links' )`&mdash; translations: **23**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/4603)